addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -223,13 +227,11 @@ def _l10n_it_post_invoices_step_2(self, invoices):
             xml = proxy_user._decrypt_data(response['file'], response['key'])
             response_tree = etree.fromstring(xml)
             if state == 'ricevutaConsegna':
-                if invoice._is_commercial_partner_pa():
-                    to_return[invoice] = {'error': _('The invoice has been succesfully transmitted. The addressee has 15 days to accept or reject it.')}
-                else:
-                    to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id, 'success': True}
+                to_return[invoice] = {'error': _('The invoice has been succesfully transmitted. The addressee has 15 days to accept or reject it.')}
```

Comment:
Why?  The logic above was correct.  If the partner is from the government, he can still refuse within 15 days, otherwise not.